content/contribute/terms-and-conditions.md

```diff
@@ -7,3 +7,43 @@ linked:
 ---
 
 ## HCA Data Contributor Terms and Conditions
+
+### Preliminary terms and conditions for data use and sharing
+
+Version: 
+
+As soon as the official Human Cell Atlas terms and conditions for data contribution are finalised and reviewed by our legal authorities, you will be asked to agree to them. In that instance, we will delete all data you have already submitted to the Human Cell Atlas Data Coordination Platform.
+
+
+1. All data submitted to the Human Cell Atlas Data Coordination Platform will be made
+available for unrestricted public view, redistribution, and reuse. *
+
+2. All the data collaborators’ names and email addresses provided with the data will be
+available for unrestricted public view, redistribution, and reuse. *
+
+3. It is my responsibility as a data contributor to gain permission from my data
+collaborators in order to share their names and email addresses with the Human Cell Atlas
+Data Coordination Platform.
+
+4. It is my responsibility as a data contributor to ensure that the data is accurate and
+complete and to inform the Data Coordination Platform team of any updates, errors, or
+omissions in a timely manner. *
+
+5. It is my responsibility as a data contributor to ensure my data complies with the ethical,
+legal, and regulatory norms and standards of the country it was generated in, as well as
+applicable international norms and standards. In particular, I, as data contributor, must not
+include personal or confidential data unless appropriate consents and/or approvals have
+been obtained. *
+
+6. All data released by the Human Cell Atlas Data Coordination Platform will be released
+with an associated CC­BY license https://creativecommons.org/licenses/by/4.0/. This
+license allows users of the data to share, copy, and adapt the data, and obliges them to
+credit the data contributors when they redistribute the results. *
+
+7. I will be given the opportunity to read and agree to the official terms and conditions as
```

Comment:
I think the asterisks are marking statements that require the answer to be "I agree"? If so, we should add a statement like:

* = "I agree" required